src/workers/backgroundListeners.js

```diff
@@ -80,13 +80,15 @@ export const setNotifeeBackgroundListener = () => {
   try {
     notifee.onBackgroundEvent(async ({ type, detail }) => {
       const { notification } = detail;
+      console.debug('Notification event on background: ', '<add notification details here>');
```

Comment:
"Add notification details here"? Or even remove the log

Reply:
Ok. Adding to the patch.

Reply:
Removed at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/268/commits/f23bac0296e563e0129150f3c87bfc0f9b936b79